docs/pages/x/api/date-pickers/pickers-action-bar.json

```diff
@@ -5,7 +5,7 @@
         "name": "arrayOf",
         "description": "Array&lt;'accept'<br>&#124;&nbsp;'cancel'<br>&#124;&nbsp;'clear'<br>&#124;&nbsp;'today'&gt;"
       },
-      "default": "`['cancel', 'accept']` for mobile and `[]` for desktop"
+      "default": "`[]` for Desktop Date Picker and Desktop Date Range Picker\n- `['cancel', 'accept']` for all other Pickers"
```

Comment:
Why the dash in the second line?

```suggestion
      "default": "`[]` for Desktop Date Picker and Desktop Date Range Picker.\n`['cancel', 'accept']` for all other Pickers."
```

Reply:
To have the following JSDoc preview.
Do you have suggestions on how to improve it? 🤔 
<img width="673" alt="Screenshot 2024-12-30 at 18 27 05" src="https://github.com/user-attachments/assets/c49dfd81-8bfd-4797-83a3-7f245cab26f2" />